### DISCUSSION TITLE
[Question] - Transferring MAS from laptop to computer, (Google Drive)

### DISCUSSION THREAD

**Discussion by JinSakaiM** [Monika-After-Story/MonikaModDev#8739] [QA - tech support + general] [answered]
Do I have to put my persistent somewhere in the games files? (Edit) - Do I have to download another DDLC then place the persistent somewhere in the games files?

**Comment by JinSakaiM** [comment]
When I used a flash drive, It worked, but It had data from weeks ago.

**Comment by Booplicate** [comment, marked as answer]
Check [this](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-transfer-monika-to-another-computer).
